cmd/importer/db.go

```diff
@@ -136,13 +137,24 @@ func genColumnData(table *table, column *column) (string, error) {
 	if _, ok := table.uniqIndices[column.name]; ok {
 		incremental = true
 	}
+	if incremental {
+		incremental = uint32(rand.Int31n(100))+1 <= column.data.probability
+		// If incremental, there is only one worker, so it is safe to directly access datum.
```

Comment:
But `incremental` will also be true when this column is `unique`, in this case, there may be other workers.

Reply:
Updated, PTAL